test/test_helper/workspace.bash

```diff
@@ -9,10 +9,23 @@ function add_new_workspace() {
 	echo "${TMP_DIR}"
 }
 
+function add_dir_with_ws() {
+	TEST_DIR="$1"
+	TMP_DIR=$(mktemp -d)
+	cp -r "$TEST_DIR"/* "$TMP_DIR"
+
+	YAML="$TMP_DIR"/"$2"
+	WS_DIR=$(dirname "$YAML")
+	WS_NAME=$(basename $YAML .yml)
+	mv "$YAML" "$WS_DIR"/docker-compose.yml
+	tt workspace add "$WS_NAME" "$WS_DIR"
+}
+
 function add_ws_from_yaml() {
 	YAML=$1
 	WS_NAME=$(basename $YAML .yml)
 	TMP_DIR=$(mktemp -d)
+	cp -r "$(dirname $YAML)"/* "$TMP_DIR"
```

Comment:
```suggestion
```

i dont think we need this here anymore, the `add_dir_with_ws` is a separate test scenario use case

 (my `tt_ip` tests have the yml files all in one folder, so this would copy in all the yamls for each one.. no bad effect, but still)